.circleci/config.yml

```diff
@@ -203,10 +218,15 @@ jobs:
     steps:
       - checkout
       - run: brew install swiftlint
-      - run: swiftlint lint --strict
+      - run: swiftlint lint --strict --reporter junit | tee result.xml
```

Comment:
Any reason to prefer `| tee` to `>`?

Reply:
I think it make sense to report logs to the CI instead of silently redirect to the file.